paasta_tools/cli/cmds/local_run.py

```diff
@@ -754,6 +768,35 @@ def run_docker_container(
         healthcheck_only or healthcheck
     ) and healthcheck_mode is not None
 
+    for container_mount_path, secret_content in secret_volumes.items():
+        # Create local temp file in a hidden folder in cwd
+        temp_secret_folder = os.path.join(os.getcwd(), ".secret_volumes/")
+        temp_secret_filename = os.path.join(temp_secret_folder + str(uuid.uuid4()))
```

Comment:
just checking: was this meant to be
```suggestion
        temp_secret_filename = os.path.join(temp_secret_folder, str(uuid.uuid4()))
```